hail/python/test/hail/ggplot/test_ggplot.py

```diff
@@ -88,3 +88,16 @@ def test_scale_color_manual():
     pfig = fig.to_plotly()
 
     assert set([scatter.marker.color for scatter in pfig.data]) == colors
+
+
```

Comment:
You need to change `geom_bar` to add the `weight` argument right? And then add documentation of what it does?

Reply:
I'm wrong, handled in stat